tests/test_interface_ospf.rb

```diff
@@ -21,6 +21,8 @@
 
 # TestInterfaceOspf - Minitest for InterfaceOspf node utility class.
 class TestInterfaceOspf < CiscoTestCase
+  @feature = 'interface_ospf'
```

Comment:
@chburket We still need to mark the YAML support as well - the hope here was to move a bit more back in the direction of having the YAML serve as the source of truth.

@mikewiebe I prefer `@must_support` but `@skip_unless_supported` is OK by me too.